sass/sass_wrapper.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * A Sass compiler wrapper that supports bazel persistent worker protocol.
+ * 
+ * Bazel can spawn a persistent worker process that handles multiple invocations. It
+ * will also be backwards compatible with being run as a normal single-invocation command.
+ */
+const {debug, log, runAsWorker, runWorkerLoop} = require('./worker');
+const sass = require('sass');
+const fs = require('fs');
+
+// Equivalent of running node with --expose-gc
+// but easier to write tooling since we don't need to inject that arg to
+// nodejs_binary
+if (typeof global.gc !== 'function') {
+  require('v8').setFlagsFromString('--expose_gc');
+  global.gc = require('vm').runInNewContext('gc');
+}
+
+function main(args) {
+  if (runAsWorker(args)) {
+    debug('Starting Sass compiler persistent worker...');
+    runWorkerLoop(runOneBuild);
+    // Note: intentionally don't process.exit() here, because runWorkerLoop
+    // is waiting for async callbacks from node.
+  } else {
+    debug('Running a single build...');
+    if (args.length === 0) throw new Error('Not enough arguments');
+    if (args.length !== 1) {
+      console.error('Expected one argument: path to flagfile');
+      return 0;
+    }
+    // Bazel worker protocol expects the only arg to be @<path_to_flagfile>.
+    // When we running a single build, we remove the @ prefix and read the list of
+    // actual arguments line by line.
+    const configFile = args[0].replace(/^@+/, '');
+    const configContent = fs.readFileSync(configFile, 'ascii').trim();
```

Comment:
Why ASCII and not UTF-8?

Reply:
Using UTF-8 caused build errors in Buildkite (but not in my local machine), I wasn't sure why:
https://buildkite.com/bazel/rules-sass/builds/257

Reply:
This does seem a bit strange to me. Like something that seems fine now, but breaks in an unexpected way once it's in the wild. Would anyone on the bazel team know more about the file encoding in different environments? If the encoding depends on the environment, I could see this being a problem.

Reply:
It is passing now... I had no idea, but changed to 'utf-8' instead.